core/services/ardupilot_manager/typedefs.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+
+class SITLVehicle(str, Enum):
```

Comment:
This is not a vehicle configuration, it's a frame.

Reply:
Changed.